### PR TITLE
JSTL - InstructorStudentRecordsAjax

### PR DESCRIPTION
**Pull request by wkurniawan07**
Finally migrated this thing.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Just realized I never asked anyone for review. @bamboo3250 hope you don't mind!

**Comment by wkurniawan07** [comment]
@bamboo3250 I hope you can review this tomorrow!

**Comment by bamboo3250** [comment]
omg, sorry, I'll review it now :sob:

File: src/main/webapp/WEB-INF/tags/instructor/studentRecords/resultsTable.tag [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                                    	Question ${response.questionNumber}: <span class="text-preserve-space">${response.questionText}${response.questionMoreInfo}</span>
+                                    </div>
+                                    <div class="panel-body">
+                                        <div style="clear:both; overflow: hidden">
+                                            <%-- Note: When an element has class text-preserve-space, do not insert any HTML spaces --%>
+                                            <div class="pull-left text-preserve-space">${response.responseText}</div>
+                                        </div>
+                                        <c:if test="${not empty response.responseComments}">
+                                        	<ul class="list-group" id="responseCommentTable-${fbIndex}-${personIndex.index + 1}-${qnIndex.index + 1}-${viewType}" style="margin-top:15px;">
+                                        		<c:forEach items="${response.responseComments}" var="responseComment">
```
Comment (bamboo3250) [inline src/main/webapp/WEB-INF/tags/instructor/studentRecords/resultsTable.tag]: I saw a few tabs used here. Please use space instead.

File: src/main/java/teammates/ui/template/FeedbackResponseRow.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                                                                              personType + "-" + personIndex
+                                                                                         + "-session-" + fbIndex);
+        if (personType.equals("recipient")) {
+            this.responseText = response.getResponseDetails().getAnswerHtml(questionDetails);
+        } else if (personType.equals("giver")) {
+            this.responseText = results.getResponseAnswerHtml(response, question);
+        }
+        this.responseComments = new ArrayList<FeedbackResponseComment>();
+        List<FeedbackResponseCommentAttributes> frcs = results.responseComments.get(response.getId());
+        if (frcs != null && frcs.size() > 0) {
```
Comment (bamboo3250) [inline src/main/java/teammates/ui/template/FeedbackResponseRow.java]: Maybe use !frcs.isEmpty() here?

**Comment by bamboo3250** [comment]
After fixing all of them, feel free to merge :+1:

File: src/main/java/teammates/ui/template/FeedbackResponseRow.java (hunk 2 above)
Comment (wkurniawan07) [inline src/main/java/teammates/ui/template/FeedbackResponseRow.java]: Turns out the null check is sufficient, so I'm keeping that and removed the size check.

**Comment by wkurniawan07** [comment]
@bamboo3250 fixed both. Maybe one last check or just set this to `toMerge`?

**Pull request merged**